cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/local/DataStorageImpl.kt

```diff
@@ -169,6 +179,10 @@ private class DataStorageImpl(
         return preference.getString(LOCAL_STATE, null)
     }
 
+    override fun updateLocalDataVersion() {
```

Comment:
I don't see the need to implement this method. 

Whichever class is using `DataStorage` can simply set the value to be the hardcoded one.